region/crateria/west.json

```diff
@@ -1628,25 +1647,58 @@
                   ]
                 },
                 {
-                  "name": "Morph Bomb Gauntlet E-Tank Room (Out)",
-                  "notable": true,
-                  "requires": [],
+                  "name": "Morph Bomb Gauntlet E-Tank Room (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "canCarefulJump",
+                      {"acidFrames": 35}
+                    ]}
+                  ],
                   "obstacles": [
                     {
                       "id": "A",
-                      "requires": ["h_canUseMorphBombs"],
-                      "devNote": "FIXME health?"
+                      "requires": [
+                        "h_canUseMorphBombs",
+                        {"or":[
+                          "canTrickyJump",
+                          {"and":[
+                            "canCarefulJump",
+                            "canBombHorizontally",
+                            {"acidFrames": 35}
+                          ]},
+                          {"acidFrames": 100}
+                        ]}
+                      ]
                     },
                     {
                       "id": "B",
-                      "requires": ["h_canUseMorphBombs"]
+                      "requires": [
+                        "h_canUseMorphBombs",
+                        {"or":[
+                          {"and":[
+                            "canTrickyJump",
+                            "canStaggeredWalljump"
```

Comment:
A note about spinjumping into blocks broken by bombs would be helpful here.  That and morphing quickly at the right height is how you avoid damage at high level.